deny.toml

```diff
@@ -0,0 +1,144 @@
+# This section is considered when running `cargo deny check advisories`
```

Comment:
Is this mostly the default? How do we keep this updated when new options are added upstream?

Reply:
These rarely / never change as it's type of check. If there are new types of check then it is a feature and dependabot will pester about upgrading the action and it is then when you can discover any new type of checks - 

Nonetheless RustSec advisory categories may change but it is covered under advisories regardless.